x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots/open_url.ts

```diff
@@ -30,14 +31,13 @@ export const openUrl = async (
       logger
     );
   } catch (err) {
+    apm.captureError(err);
```

Comment:
I believe that I decided to capture it explicitly because the error that is thrown afterwards is eventually caught by a handler to prevent a failed reporting job.